app/auth/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+Create the Blueprint.
+
+Create and import the views to be used for the Blueprint
+"""
+from flask import Blueprint
+
+authe = Blueprint('authe', __name__)
```

Comment:
I have changed the name to `authentication` instead to better describe what is being done.